src/imageio.cs

```diff
@@ -901,6 +901,734 @@ interface CGImageProperties {
 		[Mac (10, 13), iOS (11,0), TV (11,0), Watch (4,0)]
 		[Field ("kCGImagePropertyFileContentsDictionary")]
 		NSString FileContentsDictionary { get; }
+		
+		[Mac (10, 9)][iOS (11,0)]
+		[Field ("kCGImagePropertyOpenEXRDictionary")]
+		NSString OpenEXRDictionary { get; }
```

Comment:
3 letters acronyms should be pascal cased in .net -> `Exr` (and the typo check will likely fail)

Reply:
Agreed